app/src/main/java/com/github/se/studybuddies/ui/topics/TopicResources.kt

```diff
@@ -140,6 +181,22 @@ fun TopicResources(
       }
 }
 
+@Composable
+fun ResourceImage(image: Uri) {
+  Log.d("Topics", "current image is $image")
+  Row(
+      modifier = Modifier.fillMaxWidth().padding(8.dp),
+      verticalAlignment = Alignment.CenterVertically,
+      horizontalArrangement = Arrangement.Center) {
+        Image(
+            painter = rememberAsyncImagePainter(image.toString()),
+            contentDescription = stringResource(R.string.image_resource),
+            modifier =
+                Modifier.size(200.dp).clip(RoundedCornerShape(20.dp)).testTag("chat_message_image"),
```

Comment:
the image appears quite big and in the middle of my screen, making it feel like we lose a lot of useful space, is this intentional ?

Reply:
Mmh, I was going for a sort of "feed"-looking page, where there might later be also pdfs, links, etc...